package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please remove this file as package.json covers it

Reply:
done